src/timessquare/main.py

```diff
@@ -39,54 +39,42 @@
     name=config.logger_name,
 )
 
-app = FastAPI()
-"""The main FastAPI application for times-square."""
-
-# Define the external routes in a subapp so that it will serve its own OpenAPI
-# interface definition and documentation URLs under the external URL.
-external_app = FastAPI(
-    title="times-square",
-    description=metadata("times-square").get("Summary", ""),
+app = FastAPI(
+    title="Times Square",
+    description=Path(__file__).parent.joinpath("description.md").read_text(),
     version=metadata("times-square").get("Version", "0.0.0"),
+    openapi_url=f"{config.path_prefix}/openapi.json",
+    docs_url=f"{config.path_prefix}/docs",
+    redoc_url=None,
```

Comment:
Why no `redoc_url`? As mentioned in the other review, I think the redoc output format is generally more useful than the docs output format.